src/main/java/com/khoros/twitterapp/TwitterAuthorization.java

```diff
@@ -0,0 +1,59 @@
+package com.khoros.twitterapp;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import org.hibernate.validator.constraints.NotBlank;
+import twitter4j.conf.*;
```

Comment:
don't use *

Reply:
Import statement not in use. I just removed it.